libs/vm-modules/src/ml/model/model_estimator.cpp

```diff
@@ -65,8 +65,8 @@ ModelEstimator &ModelEstimator::operator=(ModelEstimator &&other) noexcept
 ChargeAmount ModelEstimator::LayerAddDense(Ptr<String> const &layer, SizeType const &inputs,
```

Comment:
there still seems to be changes in this file. I don't understand why you need to modify the estimators at all?